crates/matrix-sdk-base/src/room/room_info.rs

```diff
@@ -464,6 +464,13 @@ pub struct RoomInfo {
     /// more accurate than relying on the latest event.
     #[serde(default)]
     pub(crate) recency_stamp: Option<u64>,
+
+    /// A timestamp remembering when we observed the user accepting an invite on
+    /// this current device.
+    ///
+    /// This is useful to remember if the user accepted this a join on using
+    /// this specific client.
```

Comment:
Something's off here, needs rewording I reckon.

Reply:
Just a leftover from the previous version of this sentence.